src/bind/input/input.cpp

```diff
@@ -51,9 +52,12 @@ void init_input(py::module_ &m) {
       .def("has_homogeneous_locations",
            &vroom::Input::has_homogeneous_locations)
       .def("has_homogeneous_profiles", &vroom::Input::has_homogeneous_profiles)
+      .def("has_homogeneous_costs", &vroom::Input::has_homogeneous_costs)
       // .def("vehicle_ok_with_job", &vroom::Input::vehicle_ok_with_job)
       .def("_solve", &vroom::Input::solve, "Solve problem.",
-           py::arg("exploration_level"), py::arg("nb_threads") = 1,
+           py::arg("nb_searches"),
+           py::arg("depth"),
```

Comment:
Looks like `vroom::Input` and `vroom::input::solve` have been changed a bit. Do we have documentation for `apply_TSPFix`, `nb_searches` and `depth`? I am guessing the former is experimental, and the two latter are core parameters.

Reply:
Yes, the `TSPFix` operator is definitely experimental. The reasons for disabling it by default are discussed in https://github.com/VROOM-Project/vroom/issues/1013.

The `nb_searches` and `depth` parameters are internal parameters that we don't really expose outside the C++ functions. When using from command-line, there is a "meta" parameter that wraps the various trade-offs we offer `  -x, --explore arg  exploration level to use (0..5) (default: 5)`. The way this translates to the above is defined in https://github.com/VROOM-Project/vroom/blob/31bace492abfd97155faf91c20b4300dfbfcdcec/src/structures/cl_args.cpp#L78-L90.

Having a simple knob to use on a scale from 0 to 5 is actually convenient from a user perspective because everything is already tuned internally and you don't have to understand what the underlying parameters are about. Maybe that's what we'd also like to expose from pyvroom to offer the same interface?

Reply:
I think have the parameters consistent would be a good idea, yeah.
For now I'll do the translation manually on the Python side to get something working.

Reply:
What we could do if it helps is to keep an overload of `Input::solve` with the old `exploration_level` signature. This would trivially call the new version with the expected conversion for the new parameters. Then you'd only have to keep exposing that one for the bindings?

Reply:
If we can have that, that would be great, yes.

Reply:
@jonathf done in https://github.com/VROOM-Project/vroom/pull/1197, let me know if that's OK and I'll merge.